tasks/lib/phantomas.js

```diff
@@ -506,7 +506,7 @@ Phantomas.prototype.getImages = function() {
 
   return _.sortBy( files, _.bind( function( file ) {
     return +file.match(
-      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d-(\d*).png$/
+      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d-\d\d-\d\d-(\d*).png$/
```

Comment:
It generated images like:

- `screenshot-2014-11-30T14:06:31-41.png`
- `screenshot-2014-11-30T14:06:31-767.png`
- `screenshot-2014-11-30T14:06:31-1124.png`

I'm grabing everything behind the dash ( which was the time passed by, or ) and use this for sorting.

```
  return _.sortBy( files, _.bind( function( file ) {
    return +file.match(
      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d-(\d*).png$/
    )[ 1 ];
  }, this ) );
```

The binding is not needed - but I think it makes sense. :)